chain-patrol-ts/package.json

```diff
@@ -38,8 +38,12 @@
     "forta-agent-tools": "^3.2.10",
     "lru-cache": "^10.0.1"
   },
+  "overrides": {
+    "semver": "^7.5.3"
+  },
```

Comment:
what was this about?

Reply:
There was an error when I was trying to publish the bot ([this one](https://github.com/advisories/GHSA-c2qf-rxjj-qqgw) specifically). After investigating it, this ([here](https://stackoverflow.com/questions/76538757/how-to-approach-and-fix-npm-security-issues-semver-vulnerable-to-regular-expre)) was a solution I came across online to resolve it, I tried it, and it solved the issue.

Reply:
I think it's also fixed by updating `nodemon` to `^3.0.1`